modules/sharedaddy/sharing.js

```diff
@@ -6,7 +6,7 @@ if ( sharing_js_options && sharing_js_options.counts ) {
 		twitter_count : {},
 		get_counts : function() {
 			var facebookPostIds = [],
-				https_url, http_url, url, urls, id, service, service_url, path_ending;
```

Comment:
`url` shouldn't have been removed here - it's still used below in the `https_url`/`http_url` assignment and in the `for` loop iterating over `WPCOM_sharing_counts`.